### ISSUE TITLE
`error: package bgrabitmappack.lpk is only for runtime` while installing packages to Lazarus

### ISSUE THREAD

**Issue by veksha** [bogen85/CudaText_up_py#1]
OS: Mac

Hi. thanks for providing this script! it helped me to download CudaText packages and install it into Lazarus.
But not without a hassle.

1. doing `./cudaup.sh -g` worked nicely!
2. but then I needed to install these packages for Lazarus.
I did the command `./cudaup.sh -p -l /Applications/Lazarus/`

and got error:
![image](https://user-images.githubusercontent.com/275333/181868149-a3d23aca-5282-4a63-95df-00d5b0d991da.png)

and the script stopped.
Looking at the python code I somehow figured out that I need to edit `cudaup.packets` file and remove packages that are "only for runtime":

Changed these lines:

![image](https://user-images.githubusercontent.com/275333/181867373-c572fa4e-a5a5-4698-b8ab-25ba981cf7c1.png)

to these 4 lines:

![image](https://user-images.githubusercontent.com/275333/181867323-bc703250-33f2-4199-beae-0966bc4cb310.png)

And after that the script succeeds! Then I've just opened CudaText project in Lazarus and compile it.

Just letting you, guys, @bogen85 @Alexey-T , know that this script can be improved. (not by editing `cudaup.packets`, but by fixing python code)

Thanks again.

**Comment by Alexey-T** [comment]
Yes, some packages are runtime only and we must not install them to IDE. I don't know how to change cudaup (original).

**Comment by veksha** [comment]
>  I don't know how to change cudaup (original).

original (not python version) had no such bug, as far as I remember.

**Comment by bogen85** [comment]
I can test this on MacOS. I'll see what I can do.

**Comment by bogen85** [comment]
@veksha I'll also take a look at the change you did for cudaup.packets
